javascript/ql/lib/semmle/javascript/linters/ESLint.qll

```diff
@@ -60,22 +60,22 @@ module ESLint {
   }
 
   /** An `.eslintrc.yaml` file. */
-  private class EslintrcYaml extends Configuration, YamlDocument, YamlMapping {
+  private class EslintrcYaml extends Configuration instanceof YamlMapping {
```

Comment:
```suggestion
  private class EslintrcYaml extends Configuration instanceof YamlMapping, YamlDocument {
```

Things might be different if we don't force `EslintrcYaml` to be a `YamlDocument`.